starter/src/components/Locator.tsx

```diff
@@ -1,27 +1,56 @@
 import { ComponentConfig, Fields } from "@measured/puck";
+import { FilterSearch } from "@yext/search-ui-react";
+import {
+  CloudChoice,
+  CloudRegion,
+  Environment,
+  provideHeadless,
+  SearchHeadlessProvider,
+} from "@yext/search-headless-react";
+import * as React from "react";
 
 type LocatorProps = {
-  text: string;
+  apiKey: string;
 };
 
 const fields: Fields<LocatorProps> = {
-  text: {
-    label: "Text",
+  apiKey: {
+    label: "API Key",
     type: "text",
   },
 };
 
 const LocatorComponent: ComponentConfig<LocatorProps> = {
   fields,
   defaultProps: {
-    text: "Some Placeholder",
+    apiKey: "Fake Key",
   },
   label: "Locator",
   render: (props) => <Locator {...props} />,
 };
 
-const Locator: React.FC<LocatorProps> = ({ text }) => {
-  return <div>{text}</div>;
+const Locator: React.FC<LocatorProps> = (props) => {
```

Comment:
Destructure the props, either:

inline
`const Locator: React.FC<LocatorProps> = ({apiKey}) => {`

or

```
const Locator: React.FC<LocatorProps> = (props) => {
  const { apiKey, other...} = props;
```